backend/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "dist/app.js",
   "scripts": {
-    "start:dev": "npx tsc && NODE_ENV=development node build/app.js",
+    "start:dev": "npx tsc && node build/app.js",
```

Comment:
Is this intentional?

Reply:
sorry, accidentally commited - doesn't work on Windows otherwise. will revert

Reply:
> doesn't work on Windows otherwise
that's weird let's follow up